gnd/src/main/res/layout/offline_area_viewer_frag.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2020 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+    <variable
+      name="viewModel"
+      type="com.google.android.gnd.ui.offlinearea.viewer.OfflineAreaViewerViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true">
+
+    <com.google.android.gnd.ui.common.TwoLineToolbar
+      android:id="@+id/offline_area_viewer_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:elevation="@dimen/toolbar_elevation"
+      android:theme="@style/PrimaryToolbarTheme"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:title="@string/offline_area_viewer_title"
+      app:titleTextColor="@color/colorBackground" />
+
+    <FrameLayout
+      android:id="@+id/offline_area_viewer_map_frame"
+      android:layout_width="match_parent"
+      android:layout_height="200dp"
+      android:background="@color/colorGrey300"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.333"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/offline_area_viewer_toolbar"
+      app:layout_constraintVertical_bias="0.533">
+      <androidx.fragment.app.FragmentContainerView
+        android:id="@+id/map"
+        android:layout_width="140dp"
```

Comment:
Any specific reasons for hardcoding the width of the fragment? I would rather use constraints which only relies on screen ratio.